reading-pool.md

```diff
@@ -79,7 +79,7 @@ Why Decentralize?
 
 Requirements for co-ops in Canada (Federal and Provincial) (for more see [**2018 retreat playbook**](https://link.hypha.coop/retreat-playbook)):
 
-- 📄 [Registering a coop - Official Guide by Financial Services Commision of Ontario](https://www.fsco.gov.on.ca/en/coops/Documents/register_co-ops-guide.pdf)
+- 📄 [Registering a coop - Official Guide by Financial Services Commision of Ontario](http://www.fsco.gov.on.ca/en/coops/Documents/guide_coop.pdf)
```

Comment:
Not sure if this is the same document, but it seems to be can some one please confirm

Reply:
```suggestion
- 📄 [Registering a coop - Official Guide by Financial Services Commision of Ontario](https://www.fsco.gov.on.ca/en/coops/Documents/guide_coop.pdf)
```

Reply:
Not sure if same. @dcwalk ?

Reply:
Not sure, but looks similar, why not just drop the link?

Reply:
```suggestion
```